source/Octopus.Tentacle/Services/ServicesModule.cs

```diff
@@ -22,19 +21,21 @@ protected override void Load(ContainerBuilder builder)
 
             builder.RegisterType<NuGetPackageInstaller>().As<IPackageInstaller>();
 
-            // Register the script executor based on if we should
+            // Register our Halibut services
+            var allTypes = ThisAssembly.GetTypes();
+            RegisterHalibutServices<ServiceAttribute>(builder, allTypes);
+
+            //only register kubernetes services when
```

Comment:
We are still registering non-K8s services when running as a an Agent.

Should we consider registering the services explicitly that's somehow in sync with what `CapabilitiesServiceV2` returns

Reply:
Might be worth discussing with @OctopusDeploy/team-server-at-scale at some point 🤔